BaseFilter/sections/specific.txt

```diff
@@ -16872,7 +16872,7 @@ check-host.net##div[class*="-self-"]
 check-host.net##div[class*="t-"][class*="-elative"]
 /^https:\/\/check-host\.net\/r?images\/[a-z]{2,15}[0-9]{1,3}-?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png/$domain=check-host.net
 check-host.net#?#div + div[class*=" "]:has(> div > span > div:contains(/^ ?[×✕✖X☓χx⨯✗\+\＋\﹢хХΧẊẋ] ?$/))
-check-host.net#?#a[href]:contains(/[Aa][eé]za|100[%％﹪⁒]|Incogni/):upward(1)
+check-host.net#?#a[href]:contains(/[Aa][eé]z[aA]|100[%％﹪⁒]|[Ii]ncogni/):upward(1)
```

Comment:
There is a leftover https://check-host.net/ip-info. I guess the rule can be removed as it blocked by another one.

<details><summary>Screenshot:</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/33488229/18cd9dc9-bbb6-461e-80e4-426ce03e326d)

</details><br/>